testsuite/tests/ui/devel/test_devel_sections.py

```diff
@@ -0,0 +1,101 @@
+    """Creates a new section in developer portal"""
+    view = navigator.navigate(DeveloperPortalContentNewSectionView)
+    section_name = blame(request, "TestSection")
+    section_path = blame(request, "section")
+    view.create(section_name, section_path, False)
+    section_id = navigator.browser.url.split("/")[-2]
+
+    def cleanup():
+        custom_admin_login()
+        view = navigator.navigate(DeveloperPortalContentEditSectionView, section_id=section_id)
+        view.delete()
+
+    request.addfinalizer(cleanup)
+    return section_name
+
+
+@pytest.fixture()
+def dev_portal_page(login, navigator, request, dev_portal_section, custom_admin_login):
+    """Creates a new page in developer portal and assign it to section"""
+    view = navigator.navigate(DeveloperPortalContentNewPageView)
+    page_name = blame(request, "TestPage")
+    page_path = blame(request, "/test")
+
+    view.create(page_name, dev_portal_section, page_path, "<h1>Test</h1>")
+    page_id = navigator.browser.url.split("/")[-2]
+
+    def cleanup():
+        custom_admin_login()
+        view = navigator.navigate(DeveloperPortalContentEditPageView, page_id=page_id)
+        view.delete()
+
+    request.addfinalizer(cleanup)
+
+    view = navigator.new_page(DeveloperPortalContentEditPageView, page_id=page_id)
+    view.publish()
+
+    return view.get_path()
+
+
+@pytest.fixture()
+def dev_portal_group(login, navigator, request, account, dev_portal_section, custom_admin_login):
+    """Creates a new group in developer portal and assign account to this group"""
+    group_name = blame(request, "TestGroup")
+    view = navigator.navigate(DeveloperPortalGroupNewView)
+
+    def cleanup():
+        custom_admin_login()
+        view = navigator.navigate(DeveloperPortalGroupView)
+        view.delete_group(group_name)
+
+    request.addfinalizer(cleanup)
+    view.create(group_name, [dev_portal_section])
+
+    view = navigator.navigate(AccountUserGroupView, account=account)
+    view.update([group_name])
+
+
+@pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-9020")
+@pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-836")
+@pytest.mark.skipif("TESTED_VERSION < Version('2.14')")
+def test_dev_portal_sections(account, custom_devel_login, browser, testconfig, dev_portal_section, dev_portal_page,
+                             dev_portal_group, navigator):
+    """
+    Preparation:
+        - Creates a new dev portal section
+        - Creates a new dev portal page in that section
+        - Creates a new dev portal group for that section
+    Test:
+        - Login to dev portal as account with assigned group
+        - Assert that this account has access to this page
+        - Login to dev portal as account without group
+        - Assert that this account hasn't access to this page
+    """
+    custom_devel_login(account=account)
+    browser.url = testconfig["threescale"]["devel"]["url"] + dev_portal_page
```

Comment:
`dev_portal_page` should return the whole URL, not just its path. by doing this you don't need to construct whole URL in the test and you can omit `testconfig` fixture

Reply:
I can't get the whole URL from NewPageView.